CONTRIBUTING.md

```diff
@@ -43,24 +51,24 @@ install everything you need in a Python virtual environment.
 Install the required tools:
 
 ``` bash
-$ sudo apt install git python3-virtualenv
+sudo apt install git python3-virtualenv
 ```
 
 Prepare the development environment. If you are an external contributor and
 plan on submitting some changes, you will have to [fork the Checkbox repository
 first], and clone your own version locally. Otherwise:
 
 ``` bash
-$ cd ~
-$ git clone git@github.com:canonical/checkbox.git
+cd ~
+git clone git@github.com:canonical/checkbox.git
```

Comment:
I originally wrote code snippets like this, but @tang-mm pointed out this makes it harder to see what lines are commands and what lines are output from the command.

Something like:

```
$ ls
AccountsService/      git/                    snapd/
alsa/                 grub/                   snmp/
$ cd git
$ ls -l
total 0
```

is easier to read than

```
ls
AccountsService/      git/                    snapd/
alsa/                 grub/                   snmp/
cd git
ls -l
total 0
```

(I know it's not the case in this change, but there might be some commands with an output in this doc at some point)

Reply:
And I 100% agree with that, the problem is that the thing I removed is:
```
$ cd ~/checkbox/checkbox-ng
$ ./mk-venv
$ . ~/checkbox-ng/venv/bin/activate
```
In which there's no output of the commands being run, thus making it not really the case (and also a MD syntax problem).